config.c

```diff
@@ -1316,11 +1345,19 @@ int pv_config_init(char *path)
 		path = PV_PANTAVISOR_CONFIG_PATH;
 
 	printf("DEBUG: loading config from %s\n", path);
-	if (pv_config_load_config_from_file(path, &pv->config) < 0) {
+	if (pv_config_load_file(path, &pv->config) < 0) {
 		printf("FATAL: unable to parse %s\n", path);
 		return -1;
 	}
 
+	const char *policy = pv_config_get_policy();
+
+	printf("DEBUG: loading policy '%s'\n", policy);
```

Comment:
are we sure we cannot use pv_log at this point yet?

Reply:
this is previous to logserver initialization